washingtonpost/elex-solver
# Elex 2406 pre commit

## Description
Added pre-commit hooks. Also ran pre-commit against all files, updated readme and added github actions to run when PR opens.

## Jira Ticket
[elex-2406](https://arcpublishing.atlassian.net/browse/ELEX-2406)

## Test Steps
```
pip install -r requirements-dev.txt
pre-commit run --all-files
```
github actions automatically run. See below.
